
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  </head>
<body ng-app="app">
<div class="container">
        <h2>Metascape Gene List Analysis Report</h2><p/>  <a href="http://metascape.org">metascape.org</a><sup>1</sup><p/>

      <h3>Gene Lists</h3>
      User-provided gene identifiers are first converted into their corresponding H. sapiens Entrez gene IDs using the latest version of the database (last updated on 2022-01-01).  If multiple identifiers correspond to the same Entrez gene ID, they will be considered as a single Entrez gene ID in downstream analyses.   Each gene list is assigned a unique color, which is used throughout the analysis.   The gene lists are summarized in Table 1.<p/>
          <div class="panel panel-info">
            <div class="panel-heading">Table 1. Statistics of input gene lists.</div>
            <div class="panel-body"><TABLE class="table">
  <THEAD>
    <TR>
      <TH class="info">Name</TH>
      <TH class="info">Total</TH>
      <TH class="info">Unique</TH>
      <TH class="info">Color Code</TH>
    </TR>
  </THEAD>
  <TBODY>
    <TR>
      <TD>LN_meta</TD>
      <TD>260</TD>
      <TD>225</TD>
      <TD><div style="background-color:#E41A1C !important;width:80px;height:20px;"></div></TD>
    </TR>
    <TR>
      <TD>N0</TD>
      <TD>193</TD>
      <TD>171</TD>
      <TD><div style="background-color:#377EB8 !important;width:80px;height:20px;"></div></TD>
    </TR>
  </TBODY>
</TABLE></div>
          </div>
 The overlaps between these lists are shown in a <a href="http://circos.ca">Circos</a><sup>2</sup> plot (Figure 1).<p/>
                          <div class="panel panel-info">
                            <div class="panel-heading">Figure 1. Overlap between gene lists.</div>
                            <div class="panel-body">
                            <table>
                                <tr>
                                    <td>
                                        <img src="./Overlap_circos/CircosOverlapByGene.png" style="width:500px;">
                                    </td>
                                </tr>
                                <tr>
                                    <td align='center'>
                                        
                                <a href='./Overlap_circos/CircosOverlapByGene.svg' title='download SVG file'>
                                    <img class='link' src='icon/SVG48.png' height="48" width="48" >
                                </a>
                                    </td>
                                </tr>
                            </table>

                                </div>
                          </div><p/>



<h3>Protein-protein Interaction Enrichment Analysis</h3>
For each given gene list, protein-protein interaction enrichment analysis has been carried out with the following databases: STRING<sup>3</sup>, BioGrid<sup>4</sup>, OmniPath<sup>5</sup>, InWeb_IM<sup>6</sup>.Only physical interactions in STRING (physical score &gt 0.132) and BioGrid are used (<a href="http://metascape.org/blog/?p=219">details</a>).  The resultant network contains the subset of proteins that form physical interactions with at least one other member in the list.  If the network contains between 3 and 500 proteins, the Molecular Complex Detection (MCODE) algorithm<sup>7</sup> has been applied to identify densely connected network components.  


    <h3>Reference</h3></p>
    <ol style="list-style: decimal inside;">
      <li>Zhou et al., Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. Nature Communications (2019) 10(1):1523.</li>
      <li>Krzywinski M. et al. Circos: an Information Aesthetic for Comparative Genomics. Genome Res (2009) 19:1639-1645</li>
      <li>Szklarczyk D. et al. STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. (2019) 47:D607-613.</li>
      <li>Stark C. et al. BioGRID: a general repository for interaction datasets. Nucleic Acids Res. (2006) 34:D535-539.</li>
      <li>Turei D. et al. A scored human protein-protein interaction network to catalyze genomic interpretation. Nat. Methods. (2016) 13:966-967.</li>
      <li>Li T. et al. A scored human protein-protein interaction network to catalyze genomic interpretation. Nat. Methods. (2017) 14:61-64.</li>
      <li>Bader, G.D. et al. An automated method for finding molecular complexes in large protein interaction networks. BMC bioinformatics (2003) 4:2.</li>
</ol>
</div>

</body></html>